Box.V2.Test.Integration/BoxSignRequestManagerIntegrationTest.cs

```diff
@@ -56,9 +68,22 @@ public async Task CreateSignRequestAsync_ForCorrectSignRequestCreateRequest_Shou
             Assert.AreEqual(signRequestCreateRequest.ParentFolder.Id, signRequest.ParentFolder.Id);
 
             // first signer is the sender with role final_copy_reader, second is the recipient with role signer
-            Assert.AreEqual(2, signRequest.Signers.Count);
+            Assert.AreEqual(3, signRequest.Signers.Count);
             Assert.IsNotNull(signRequest.Signers[1].IframeableEmbedUrl);
 
+            var signerGroupId = "";
+            for (var i = 0; i < signRequest.Signers.Count; i++)
+            {
+                if (signRequest.Signers[i].Role == BoxSignRequestSignerRole.signer)
+                {
+                    if (string.IsNullOrEmpty(signerGroupId))
+                    {
+                        signerGroupId = signRequest.Signers[i].SignerGroupId;
+                    }
+                    Assert.AreEqual(signerGroupId, signRequest.Signers[i].SignerGroupId);
+                }
+            }
```

Comment:
foreach is preferred to avoid mutability of the index and to improve readability